app/integration_action_test.go

```diff
@@ -381,10 +381,8 @@ func TestPostActionProps(t *testing.T) {
 	require.Nil(t, err)
 	assert.True(t, len(clientTriggerId) == 26)
 
-	pchan := th.App.Srv.Store.Post().GetSingle(post.Id)
-	result := <-pchan
-	require.Nil(t, result.Err)
-	newPost := result.Data.(*model.Post)
+	newPost, err := th.App.Srv.Store.Post().GetSingle(post.Id)
+	require.Nil(t, err)
```

Comment:
Please check if `newPost` is `NotNil`. Otherwise this can cause a NPE.

Reply:
Actually, I don't think that check is needed, here, if we already check that the error is not nil, the newPost must be not nil.